config/storage.yml

```diff
@@ -1,6 +1,5 @@
-test:
-  service: Disk
-  root: <%= Rails.root.join("tmp/storage") %>
+cloudinary:
+  service: Cloudinary
 
```

Comment:
just make sure the master has those same 2 lines and if so, choose the master over tis one